api/routers.go

```diff
@@ -55,11 +55,8 @@ func ContentTypeCheckerMiddleWare(next http.Handler) http.Handler {
 	})
 }
 
-// disallow body in requests who don't need it (according to the specs)
-// A HTTP/2 bug results in a body non nil and non equals to http.NoBody in the http request
-// Therefore we need to check if the given body is empty or not
-// To do that we try to read one byte from the body and check if we have io.EOF
-func RouteWithoutJsonBody(r *http.Request, rm *mux.RouteMatch) bool {
+func RequestWithoutJsonBody(r *http.Request) bool {
```

Comment:
Since request is a parameter, the function name seems redundant to me.
Can we change RequestWithoutJsonBody(r *http.Request) to HasJsonBody(r *http.Request) ?
It appears clearer in a conditional:
```go
if HasJsonBody(r) {
 // ...
}
```